src/configs/Config.ts

```diff
@@ -55,5 +56,30 @@ export const config = {
     "redis_host": process.env.redis_host || 'obsrv-redis-headless.redis.svc.cluster.local',
     "redis_port": process.env.redis_port || 6379
   },
-  "exclude_datasource_validation": ["system-stats", "failed-events-summary"]
+  "exclude_datasource_validation": process.env.exclude_datasource_validation || ["system-stats", "failed-events-summary", "masterdata-system-stats"],
+  "telemetry_dataset": process.env.telemetry_dataset || "telemetry",
+  "table_names": {
+    "datasets": "datasets",
+    "datasources": "datasources",
+    "datasetSourceConfig": "dataset_source_config",
+    "datasetTransformations": "dataset_transformations"
+  },
+  "table_config": {
+    "datasets": {
+      "pk": "id",
```

Comment:
`pk` is a confusing word. Can we either add the description or change the variable name?